solvebio/cli/data.py

```diff
@@ -743,6 +743,8 @@ def _ls(full_path):
                 file_.last_modified, file_.object_type.ljust(8), file_.full_path
             )
         )
+        if recursive and file_.object_type == "folder":
+            _ls(file_.full_path + "/*", recursive=True)
 
     if len(files) == 0:
```

Comment:
This logic needs to be modified if this is now a recursive function. Currently, if there's any empty directory in the descendents, it prints out the "No files found..." message.

Reply:
Ah, makes sense, thanks

Reply:
Pushed a change to handle this